Source/Scene/Expression.js

```diff
@@ -940,6 +940,10 @@ define([
         return isFinite(this._left.evaluate(frameState, feature));
     };
 
+    Node.prototype._evaluateBooleanConversion = function(feature) {
+        return Boolean(this._left.evaluate(feature));
+    };
+
```

Comment:
Remove this function, it is a duplicate of below. Also remove `_evaluateAbsoluteValue`, `_evaluateCosine`, `_evaluateSquareRoot`.